examples/hacker_news/hacker_news/jobs/hacker_news_api_download.py

```diff
@@ -64,30 +60,22 @@ def hourly_download_config(start: datetime, end: datetime):
 
 download_staging_job = hacker_news_api_download.to_job(
     resource_defs={
-        **{
-            "partition_bounds": partition_bounds,
-            "hn_client": hn_api_subsample_client.configured({"sample_rate": 10}),
-        },
-        **RESOURCES_STAGING,
+        "partition_bounds": partition_bounds,
+        "hn_client": hn_api_subsample_client.configured({"sample_rate": 10}),
     },
     tags=DOWNLOAD_TAGS,
     config=hourly_download_config,
 )
 
 download_local_job = hacker_news_api_download.to_job(
     resource_defs={
-        **{"partition_bounds": partition_bounds, "hn_client": hn_snapshot_client},
-        **RESOURCES_LOCAL,
-    },
-    config={
-        "resources": {
-            "partition_bounds": {
-                "config": {
-                    "start": "2020-12-30 00:00:00",
-                    "end": "2020-12-30 01:00:00",
-                }
-            },
-        }
+        "partition_bounds": partition_bounds.configured(
+            {
+                "start": "2020-12-30 00:00:00",
+                "end": "2020-12-30 01:00:00",
+            }
+        ),
+        "hn_client": hn_snapshot_client,
```

Comment:
another example where we may want to keep the resource around in the job itself.